configurator/configurator.go

```
@@ -390,15 +390,27 @@ func NewPSKHybridTriremeWithMonitor(
 		nil,
 	)
 	// use rpcmonitor no need to return it since no other consumer for it
-	rpcmon, _ := rpcmonitor.NewRPCMonitor(
+	rpcmon, err := rpcmonitor.NewRPCMonitor(
 		rpcmonitor.DefaultRPCAddress,
 		triremeInstance,
 		eventCollector,
 	)
 
+	if err != nil {
+		log.WithFields(log.Fields{
+			"package": "configurator",
+		}).Warn("Failed to initialize RPC monitor")
+		return nil, nil, nil, nil
```

this should return the error instead of a bunch of nils.